lib/model/rwfolder.go

```diff
@@ -60,6 +60,9 @@ var (
 	activity               = newDeviceActivity()
 	errNoDevice            = errors.New("peers who had this file went away, or the file has changed while syncing. will retry later")
 	errSymlinksUnsupported = errors.New("symlinks not supported")
+	errDirHasToBeScanned   = errors.New("directory contains unexpected files, scheduling scan")
+	errDirHasIgnored       = errors.New("directory contains ignored files (see ignore documentation for (?d) prefix)")
+	errDirHasKnown         = errors.New("directory contains valid files, if it is not fixed in a subsequent pull check remote")
```

Comment:
What is this intending to say?

Reply:
This is a failed attempt to compactify the information that there are files inside the folder, which according to the db and ignores should really be there, but their parent dir is still to be deleted and this might be transient (if it isn't, it definitely is a bug). Probably this should just be a generic "directory contains files error".